frontend/src/app/result-list/index.ts

```diff
@@ -0,0 +1,3 @@
+import { ResultList } from './result-list.component';
+export *  from './result-list-entry/result-list-entry.component';
+export default ResultList;
```

Comment:
https://basarat.gitbooks.io/typescript/docs/tips/barrel.html

This index.ts is our barrel file. Instead of having imports from every single individual file in our components. We can make sure that we import from the same *AREA* instead. e.g. 

In another file you might need to import from both `../../result-list.component` and `../../result-list-entry.component`

instead, we can combine both imports to take from the overall directory: `../../result-list` to show that these imports belong to a certain feature, which in this case is the `result-list`